Description: Add top-level makefile
 Adds a makefile to make building compatible with the Debian defaults.
Author: Jake O'Shannessy <joshannessy@smokecloud.io>
---
--- /dev/null
+++ fds-6.7.4/Makefile
@@ -0,0 +1,20 @@
+FINAL_INSTALL_DIR=$(DESTDIR)/usr/bin
+INSTALLDIR=$(FINAL_INSTALL_DIR)
+pkgver=6.7.4
+
+build:
+	cd Build/impi_intel_linux_64 && . /opt/intel/oneapi/setvars.sh && ./make_fds.sh
+	cp Build/impi_intel_linux_64/fds_impi_intel_linux_64 fds-exec
+
+install:
+	mkdir -p $(INSTALLDIR)
+	echo "#!/bin/sh" > fds-script
+	echo "PROGRAM_VERSION=$(pkgver)" >> fds-script
+	echo "FDS_EXEC=fds-exec-$(pkgver)" >> fds-script
+	cat fds.sh >> fds-script
+	install fds-script "${INSTALLDIR}"/fds-$(pkgver)
+	install fds-exec "${INSTALLDIR}"/fds-exec-$(pkgver)
+
+clean:
+	cd Build/impi_intel_linux_64 && rm -f *.o *.mod fds_impi_intel_linux_64
+	rm -f fds-exec fds-script
